system/vmware-monitoring/templates/vrops-inventory-vpa.yaml

```diff
@@ -17,13 +20,14 @@ spec:
   targetRef:
     apiVersion: v1
     kind: Deployment
-    name: vrops-inventory
+    name: {{ include "vropsInventory.fullName" (list $target $root) }}
   updatePolicy:
-    updateMode: "Auto"
```

Comment:
Why changing this?

Reply:
I like to try this in QA first before setting it for real. Also, it would currently have no effect to the prod regions, but if we decide to rollout the updater everywhere, I rather want to have the rollout of the VPA controlled, instead of having the VPA waiting to fire at soon as possible